### ISSUE TITLE
imdbphp 2 no longer pulling all information

### ISSUE THREAD

**Issue by underslung** [pellcorp/opendb#89]
I noticed recent additions weren't pulling images so I went to check for an imdbphp update and found it's been restructured.  It's easy to move from v2 to v3, though.  Download and extract it into ./lib/site and symlink imdbphp2 to it, then change this in imdbphp.class.php:

//include_once("./lib/site/imdbphp2/imdb.class.php");
//include_once("./lib/site/imdbphp2/imdbsearch.class.php");
include_once("./lib/site/imdbphp2/bootstrap.php");

Then at line 118 change imdb to \Imdb\Title :

                        $imdb = new \Imdb\Title($search_attributes_r['imdb_id']);

**Comment by pellcorp** [comment]
Can you create a pull request with this change. I can then merge from ui

**Comment by underslung** [comment]
Yeah, I will when I look at it a bit more, this isn't really a complete solution, just got me up and running.  How relevant is the imdb.config.php file these days?  It also references an old class and is just getting ignored.  I edited Config.php in the new imdbphp directly.

**Comment by pellcorp** [comment]
If its ignored it sounds fine to delete it.

**Comment by pellcorp** [comment]
Did you get anywhere with this I never saw an mr

**Comment by underslung** [comment]
I mostly forgot about it. I'll try and take a look tomorrow. Whatever I ended up doing works, I've upgraded imdbphp several times with no issues.

**Comment by pellcorp** [comment]
If we can avoid soft links and just copy imdbphp into the lib for. Opendb is windows compatible so no soft link

**Comment by pellcorp** [comment]
Did you make progress with this

**Comment by pellcorp** [comment]
You could clone this repo 
Then copy your changes over top of that checked out workspace and that should give you most of the changes required for a Mr.
